web/beacon-app/src/components/common/Logo.tsx

```diff
@@ -6,7 +6,7 @@ function Logo() {
   return (
     <Link to="/" data-testid="logo">
       <div className="flex items-center space-x-2">
-        <img src={RotationalLogo} alt="Rotational Labs" className="h-12 w-12" />
+        <img src={RotationalLogo} alt="Rotational Lab" className="h-12 w-12" />
```

Comment:
Is it `Rotational Lab` or `Rotational Labs`?

Reply:
It's `Labs`, Edwin added this comment here SC-13574

Reply:
i guess it's the conflict resolution that changes it. thanks for catching that.